### PR TITLE
feat(api, cli, tests): JWT access tokens management with CLI

### PR DESCRIPTION
**Pull request by fsamin**

### CODE REVIEW

File: cli/cdsctl/access_token.go [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/ovh/cds/sdk"
+
+	"github.com/ovh/cds/cli"
+	"github.com/spf13/cobra"
```
Comment (sguiheux) [inline cli/cdsctl/access_token.go]: wrong place

**Comment by ovh-cds** [comment]
CDS Report it#8627.0 ✘  
* w-it
  * w-it ✘

File: docker-compose.yml [hunk 2]
```diff
@@ -43,6 +43,8 @@ services:
       CDS_API_ARTIFACT_LOCAL_BASEDIRECTORY: /app/artifacts
       CDS_API_AUTH_SHAREDINFRATOKEN: changeitchangeitchangeitchangeitchangeitchangeitchangeitchangeit
       CDS_API_SECRETS_KEY: changeitchangeitchangeitchangeit
+      # This is a test key, you MUST chant it
```
Comment (yesnault) [inline docker-compose.yml]: is-it possible to add the cmd to generate key here?

**Comment by ovh-cds** [comment]
CDS Report build-engine-cli-tools#8639.0 ✘  
* Build minimal
  * cdsctl linux ✘  
  * check SQL Files number ✔ 
  * Engine Build linux amd64 ✘  
  * tools/debpacker ✔ 
  * tools/os-ansible-inventory ✔ 
  * Worker linux amd64 / 386 ✔

**Comment by ovh-cds** [comment]
CDS Report ut-engine#8639.0 ✘  
* Stage 1
  * Engine Test with DB ✘

**Pull request merged**